public/js/stores/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+export { default as app } from './app';
```

Comment:
can't decide, maybe explicit is better. We'd just have to change it in two places when it's explicit.